### PR TITLE
[Docs] add function `revision` + format changes to `other-functions`

### LINKED ISSUE #2004: Document `revision` function.
Create documentation for the `revision` function. See #1833 for details on how to complete this issue.

Child issue of #1833.

### PR DESCRIPTION
**Pull request by Blargian**
Closes [#2004](https://github.com/ClickHouse/clickhouse-docs/issues/2004) as part of the [functions project](https://github.com/ClickHouse/clickhouse-docs/issues/1833).

**Adds**:
- missing `revision` function to the other-functions page.

**Updates to `other-functions.md`**:

For various functions:

- replace function names like eg. `host()` with `host` and place `host()` as part of the syntax (add if missing).
- In some places the result has `Type: `, add the type in-line to better standardise formatting.
- add headings if they are missing (eg. there is an example but it's not under the heading "example")

Updates above are not-exhaustive, there are a lot of issues with this page still...  

### Changelog category (leave one):

- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit 54f6fb89eac1843058d160b39ddcb0b851f18ef8 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/64019/54f6fb89eac1843058d160b39ddcb0b851f18ef8/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/sql-reference/functions/other-functions.md [hunk 1]
```diff
@@ -288,16 +316,28 @@ Result:
...
+
+## ignore
 
 Accepts any arguments, including `NULL` and does nothing. Always returns 0.
 The argument is internally still evaluated. Useful e.g. for benchmarks.
 
+**Syntax**
+
+```sql
+ignore(…)
```
Comment (rschu1ze) [inline docs/en/sql-reference/functions/other-functions.md:338]: Maybe `ignore(x)`?

File: docs/en/sql-reference/functions/other-functions.md [hunk 2]
```diff
@@ -497,14 +541,26 @@ Result:
 └────────────────────┘
 ```
 
-## isFinite(x)
+## isFinite
```
Comment (rschu1ze) [inline docs/en/sql-reference/functions/other-functions.md:544]: `isFinite` and friends belong into docs for "Arithmetic Functions"

File: docs/en/sql-reference/functions/other-functions.md [hunk 3]
```diff
@@ -163,34 +169,58 @@ Result:
 └────────────────┴────────────────────────────┘
 ```
 
-## visibleWidth(x)
+## visibleWidth
 
 Calculates the approximate width when outputting values to the console in text format (tab-separated).
-This function is used by the system to implement Pretty formats.
+This function is used by the system to implement [Pretty formats](../formats.mdx).
```
[the remark this answers is not in the document]
Comment (rschu1ze) [reply docs/en/sql-reference/functions/other-functions.md:175]: In l. 1793, a similar comment is wrapped in `:::note`. We should do the same here.

File: docs/en/sql-reference/functions/other-functions.md [hunk 4]
```diff
@@ -2109,6 +2300,14 @@ This function is not available in ClickHouse Cloud.
 Evaluate an external catboost model. [CatBoost](https://catboost.ai) is an open-source gradient boosting library developed by Yandex for machine learning.
 Accepts a path to a catboost model and model arguments (features). Returns Float64.
 
+**Syntax**
+
+```sql
+catboostEvaluate(path_to_model, feature_1, feature_2, …, feature_n)
```
Comment (rschu1ze) [inline docs/en/sql-reference/functions/other-functions.md:2306]: Maybe your editor changed `...` (three dots) into `…` (a single three-dot character), but we should use the former form. Please also grep if the latter symbol is used elsewhere in the docs and convert it to `...`, thanks.

**Comment by rschu1ze** [comment]
Merge conflict 😢

File: docs/ru/index.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ ClickHouse — столбцовая система управления база
 | #0    | 89354350662 | 1          | Investor Relations | 1         | 2016-05-18 05:19:20 |
 | #1    | 90329509958 | 0          | Contact us         | 1         | 2016-05-18 08:10:20 |
 | #2    | 89953706054 | 1          | Mission            | 1         | 2016-05-18 07:38:00 |
-| #N    | …           | …          | …                  | …         | …                   |
+| #N    | ...           | ...          | ...                  | ...         | ...                   |
```
Comment (rschu1ze) [inline docs/ru/index.md:18]: Formatting messed up.

File: docs/ru/index.md [hunk 6]
```diff
@@ -26,11 +26,11 @@ ClickHouse — столбцовая система управления база
 
 | Строка:     | #0                 | #1                 | #2                 | #N |
 |-------------|---------------------|---------------------|---------------------|-----|
-| WatchID:    | 89354350662         | 90329509958         | 89953706054         | …   |
-| JavaEnable: | 1                   | 0                   | 1                   | …   |
-| Title:      | Investor Relations  | Contact us          | Mission             | …   |
-| GoodEvent:  | 1                   | 1                   | 1                   | …   |
-| EventTime:  | 2016-05-18 05:19:20 | 2016-05-18 08:10:20 | 2016-05-18 07:38:00 | …   |
+| WatchID:    | 89354350662         | 90329509958         | 89953706054         | ...   |
```
Comment (rschu1ze) [inline docs/ru/index.md:29]: Formatting messed up.

File: docs/zh/index.md [hunk 7]
```diff
@@ -26,11 +26,11 @@ ClickHouse是一个用于联机分析(OLAP)的列式数据库管理系统(DBMS)
 
 | Row:        | #0                 | #1                 | #2                 | #N |
 |-------------|---------------------|---------------------|---------------------|-----|
-| WatchID:    | 89354350662         | 90329509958         | 89953706054         | …   |
-| JavaEnable: | 1                   | 0                   | 1                   | …   |
-| Title:      | Investor Relations  | Contact us          | Mission             | …   |
-| GoodEvent:  | 1                   | 1                   | 1                   | …   |
-| EventTime:  | 2016-05-18 05:19:20 | 2016-05-18 08:10:20 | 2016-05-18 07:38:00 | …   |
+| WatchID:    | 89354350662         | 90329509958         | 89953706054         | ...   |
```
Comment (rschu1ze) [inline docs/zh/index.md:29]: Formatting messed up

File: docs/zh/index.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ ClickHouse是一个用于联机分析(OLAP)的列式数据库管理系统(DBMS)
 | #0 | 89354350662 | 1          | Investor Relations | 1         | 2016-05-18 05:19:20 |
 | #1 | 90329509958 | 0          | Contact us         | 1         | 2016-05-18 08:10:20 |
 | #2 | 89953706054 | 1          | Mission            | 1         | 2016-05-18 07:38:00 |
-| #N | …           | …          | …                  | …         | …                   |
+| #N | ...           | ...          | ...                  | ...         | ...                   |
```
Comment (rschu1ze) [inline docs/zh/index.md:19]: Formatting messed up.

File: docs/en/sql-reference/functions/arithmetic-functions.md [hunk 9]
```diff
@@ -140,6 +140,60 @@ Same as `intDiv` but returns zero when dividing by zero or when dividing a minim
...
+
+    SELECT 1/0 as infimum, ifNotFinite(infimum,42)
+
+Result:
+
+    ┌─infimum─┬─ifNotFinite(divide(1, 0), 42)─┐
+    │     inf │                            42 │
+    └─────────┴───────────────────────────────┘
+
+You can get similar result by using the [ternary operator](../../sql-reference/functions/conditional-functions.md#ternary-operator): `isFinite(x) ? x : y`.
```
Comment (rschu1ze) [inline docs/en/sql-reference/functions/arithmetic-functions.md:195]: Thinking about it, we should also move `isNan` from "Other Functions" to "Arihthmetic Functions". It is kind of related to `isFinite`.

**Pull request merged**